cmd/containerd-shim-runhcs-v1/events.go

```diff
@@ -15,17 +15,19 @@ type publisher interface {
 }
 
 type eventPublisher struct {
+	namespace       string
```

Comment:
Isn't this also present here https://github.com/microsoft/hcsshim/pull/1343?

Reply:
Oh, it gets removed in the next commit. When reviewing commit by commit goes wrong